.pre-commit-config.yaml

```diff
@@ -59,14 +59,19 @@ repos:
     rev: v0.5.4
     hooks:
       - id: ruff-format
+        exclude: |
+          (?x)^(
+            crates/red_knot/resources/.*
```

Comment:
I think this should just be added to the "exclude these files from all hooks" regex at the top of this file. I doubt we want any hooks to run on them in CI.

Reply:
Good call! I took the opportunity to also remove the already-redundant exclude entry on ruff, since everything listed there is already listed in the global exclude